src/test/java/it/gov/pagopa/payment/controller/BasePaymentControllerIntegrationTest.java

```diff
@@ -367,10 +402,53 @@ private TransactionInProgress checkIfStored(String trxId) {
             Assertions.assertFalse(transactionInProgressRepository.existsById(trxCreated.getId()));
         });
 
+        // useCase 6: an error occurred when publishing authorized event, returned false
+        useCases.add(i-> configureAuthEventNotPublishedDueToError(i,IDTRXISSUERPREFIX_AUTHNOTNOTIFIEDDUETOFALSE));
+
+        // useCase 7: an error occurred when publishing authorized event, throwing error
+        useCases.add(i-> configureAuthEventNotPublishedDueToError(i,IDTRXISSUERPREFIX_AUTHNOTNOTIFIEDDUETOEXCEPTION));
+
+        // useCase 8: an error occurred when publishing confirmed event, returned false
+        useCases.add(i-> configureConfirmEventNotPublishedDueToError(i,IDTRXISSUERPREFIX_CONFIRMNOTNOTIFIEDDUETOFALSE));
+
+        // useCase 9: an error occurred when publishing confirmed event, throwing error
+        useCases.add(i-> configureConfirmEventNotPublishedDueToError(i,IDTRXISSUERPREFIX_CONFIRMNOTNOTIFIEDDUETOEXCEPTION));
+
```

Comment:
even if not introduced with this PR, could you please remove the above comment:
"//andare a verificare il messaggio è presente correttamente"